Library/Homebrew/utils/shebang.rb

```diff
@@ -32,7 +32,7 @@ def initialize(regex, max_length, replacement)
     #   rewrite_shebang detected_python_shebang, bin/"script.py"
     #
     # @api public
-    sig { params(rewrite_info: RewriteInfo, paths: T::Array[T.any(String, Pathname)]).void }
```

Comment:
It looks like `sig` validation is another victim of `module_function` (I had to revert this due to the instance use in `Formula`, but kept the type signature fix here).